packages/manager/src/components/TagsPanel/TagsPanel.test.tsx

```diff
@@ -0,0 +1,126 @@
+import { fireEvent, waitFor } from '@testing-library/react';
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+import { renderWithTheme } from 'src/utilities/testHelpers';
+
+import { TagsPanel } from './TagsPanel';
+
+const queryClient = new QueryClient();
+
+const renderWithQueryClient = (ui: any) => {
```

Comment:
Can we type this? Not sure if `JSX.Element` is the best or not

Also, should we move this to `packages/manager/src/utilities/testHelpers.tsx`?

```suggestion
const renderWithQueryClient = (ui: JSX.Element) => {
```

Reply:
ah yes i left this one out. sure thing - we can be more specific too using

`React.ReactElement<TagsPanelProps>`

Reply:
also moved Tag there - makes sense!